python/src/ngraph/runtime.py

```diff
@@ -16,31 +16,42 @@
 """Provide a layer of abstraction for the ngraph++ runtime environment."""
 import logging
 from typing import Dict, List, Union
+from enum import Enum
 
 import numpy as np
 
-from ngraph.impl import Function, Node, Shape, serialize, util
+from ngraph.impl import Function, Node, Shape, PartialShape, serialize, util
 from ngraph.impl.runtime import Backend, Executable, Tensor
 from ngraph.utils.types import get_dtype, NumericData
 from ngraph.exceptions import UserInputError
 
 log = logging.getLogger(__name__)
 
 
-def runtime(backend_name='CPU'):  # type: (str) -> 'Runtime'
+class BackendMode(Enum):
+    """DYNAMIC mode enables backend's wrapper which supports dynamic shapes."""
+
+    STATIC = 0
+    DYNAMIC = 1
+
+
+def runtime(backend_name='CPU', mode=BackendMode.DYNAMIC):  # type: (str, BackendMode) -> 'Runtime'
```

Comment:
When ngraph backend is created it's static by default.
Dynamic mode as a default value can cause problems in the future.

Reply:
done